src/main/resources/logback.xml

```diff
@@ -56,9 +56,9 @@
         <appender-ref ref="METRICS" />
     </root>
 
-    <logger name="io.iohk.ethereum.network.rlpx.RLPxConnectionHandler" level="INFO" />
+    <logger name="io.iohk.ethereum.network.rlpx.RLPxConnectionHandler" level="DEBUG" />
     <logger name="io.iohk.ethereum.blockchain.sync.SyncController" level="INFO" />
-    <logger name="io.iohk.ethereum.network.PeerActor" level="INFO" />
+    <logger name="io.iohk.ethereum.network.PeerActor" level="DEBUG" />
```

Comment:
We also have some bothering logs of all the messages sent, should we remove them? Wdyt @mirkoAlic @mmrozek ?

Reply:
I think we should re-think our log polices, IMHO:
- info: Display logs that are meaningful for someone that runs the client and more a less wants to see what is going on behind the scene.
- debug: Log everything you could
By following those definitions, i think our issue is that PeerActor or network in general is no logging relevant information at  `info` level.

Reply:
I agree, but I think the only thing that we can do about this for now is pay closer attention to the logs changed or added on new PRs (till maybe someone starts using Mantis and complains about something like that). We should probably better define what a client would like to see :thinking:

Back to the question at hand, I don't see the logs of all the messages sent/received to be so useful (we haven't required them till now), or at least we should keep them in a single line for those that don't use the JSON formatting (I don't do so locally)

Reply:
I agree with @ntallar. I don't remember when we needed logs from PeerActor and when we will be sure after merge that node doesn't receive any message we won't need it anymore also ;)